### DISCUSSION TITLE
Name of 3rd SMBX Asset Pack

### DISCUSSION THREAD

**Discussion by LoveBodhi** [Wohlstand/TheXTech#433] [Polls]
There're too many name ideas for the 3rd SMBX Asset Pack.

Vote the below. Will be Finalized for the most votes.

**Comment by Wohlstand** [comment]
Some facts:
- "SMBX Revived" is a codename of SMBX2 (the name of the private repository)
- "SMBXpanded" is the name of Sednaiur's assets package where he utilised unused item slots of the SMBX64 range.

**Comment by LoveBodhi** [comment]
So let's choose SMBX+, which is the winner.
Since the name has been finalized, discussion is closed.
